LorafyAPI/Services/MQTTBackgroundService.cs

```diff
@@ -1,22 +1,28 @@
-﻿using uPLibrary.Networking.M2Mqtt;
+﻿using LorafyAPI.Services;
+using System.Text.Json;
+using uPLibrary.Networking.M2Mqtt;
 using uPLibrary.Networking.M2Mqtt.Messages;
 
 public class MQTTBackgroundService : BackgroundService
 {
     private readonly IConfiguration _configuration;
     private readonly ILogger<MQTTBackgroundService> _logger;
     private readonly MqttClient _client;
+    private string _json { get; set; }
```

Comment:
Please don't make this global, it should be a local variable in client_MqttMsgPublishReceived. Like it was before.